### DISCUSSION TITLE
twitter.com: privacy

### DISCUSSION THREAD

**Discussion by stm-x** [uBlockOrigin/uAssets#18544] [Q&A] [answered]
I see this in the uBO logger, is this a tracker?

easyprivacy seems to have the following filters, but in different directories.
`||twitter.com/1.1/jot`
`||twitter.com/1.1/promoted_content/log.json`

![image](https://github.com/uBlockOrigin/uAssets/assets/86018494/5e6cc3b5-0620-4114-a909-ffc780222c2d)
![image](https://github.com/uBlockOrigin/uAssets/assets/86018494/62b4f9b2-b73e-40e2-b5a9-ea2685187024)

**Comment by partingscientist** [comment]
Kindly cc-ing @ryanbr, I hope you don't mind. Apologies otherwise.

**Comment by partingscientist** [comment, marked as answer]
https://github.com/easylist/easylist/commit/7dc3cba8fb86eb4329913c18129d8a3903b535fb

**Comment by stm-x** [comment]
Thank you!
